.github/workflows/main.yml

```diff
@@ -25,19 +35,57 @@ jobs:
 
       - name: Install
         run: npm ci
+        working-directory: ./main
+
+      - name: Find contracts package in SDK
+        id: contracts_version
+        run: |
+          echo `npm list @unfinishedlabs/contracts --verbose`
+          export version=`npm list @unfinishedlabs/contracts --silent | sed -En 's/.*contracts@0.0.0-(\w+)\s.*$/\1/p'`
+          export result=`git ls-remote https://github.com/LibertyDSNP/contracts.git | grep -m 1 $version | cut -c 1-40`
+          echo "contract_version=$result" >> $GITHUB_ENV
+        shell: bash
+        working-directory: ./main
+
+      - name: Check result
+        run:  echo "${{ env.contract_version }}"
```

Comment:
Should we take this step out? Or, if we leave it in, could we make the message more descriptive, like "Using contract version: ${{ env.contract_version }}"?

Reply:
yes my mistake. Meant to remove.